target/linux/mediatek/dts/mt7981b-cudy-m3000-v1.dts

```diff
@@ -127,18 +126,18 @@
 		reg = <0>;
 
 		spi-max-frequency = <52000000>;
-		spi-tx-buswidth = <4>;
-		spi-rx-buswidth = <4>;
+		spi-tx-bus-width = <4>;
+		spi-rx-bus-width = <4>;
+
+		mediatek,nmbm;
+		mediatek,bmt-max-ratio = <1>;
+		mediatek,bmt-max-reserved-blocks = <64>;
 
 		partitions {
 			compatible = "fixed-partitions";
 			#address-cells = <1>;
 			#size-cells = <1>;
 
-			mediatek,nmbm;
-			mediatek,bmt-max-ratio = <1>;
-			mediatek,bmt-max-reserved-blocks = <64>;
-
```

Comment:
Did you or someone else test this change on a device?

Does the device need nmbm?

Reply:
@hauke according original SDK based on OpenWrt the devices needs nmbm.

Reply:
The OEM firmware enables nmbm, so we need it.